### ISSUE TITLE
Add notification loading to PR comment

### ISSUE THREAD

**Issue by katia-sentry** [codecov/engineering-team#553]
problem: the PR comment report doesn't appear until the first notification (Codecov worker sending information back to GH) is ready.
solution: Add a simple indicator to the comment on the PR saying "Processing". Then update when we have data.

**Comment by katia-sentry** [comment]
@codecovdesign @aj-codecov What do you think? do we need a better copy that "Processing" here? Maybe "Processing report"?

**Comment by scott-codecov** [comment]
I currently have it as:

> We're currently processing your upload.  This comment will be updated when the results are available.

**Comment by katia-sentry** [comment]
@aj-codecov @codecovdesign can you confirm ^

**Comment by aj-codecov** [comment]
@scott-codecov Copy is perfect!

**Comment by scott-codecov** [comment]
I rolled this back due to an issue that a customer was seeing.  We can see in this log entry that the initial PR comment was made and the comment ID was returned: https://l.codecov.dev/UKZHN3  But when we look in the database the `pulls.commentid` column for that PR is null.  We use the presence of that `commentid` to determine whether we've already made a comment for that PR and so we were continuing to send the "We're currently processing your upload" comment.

Need to figure out why that `commentid` could possibly be null in this case.
